drivers/max31865/dist/genlut.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2025 David Picard
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import argparse
+import numpy as np
+
+# Create a custom 4-column LUT:
+# col 0: ADC code
+# col 1: temperature in µ°c
+# col 2: slope of a piece-wise linear interpolation
+# col 3: offset of a piece-wise linear interpolation
+
+parser = argparse.ArgumentParser(prog='genlut.py',
+                                 description="Generate a header to define a custom lookup"
+                                 " table in the current directory.")
+parser.add_argument("LUTNAME", help="Suffix of the LUT name."
+                    " For example, if LUTNAME is 'mylut', the header"
+                    " file will be named 'max31865_mylut.h', the"
+                    " LUT variable will be named 'max31865_mylut'"
+                    " and the number of lines in the LUT will be named 'max31865_mylut_numlines'.")
```

Comment:
There is no generated variable `max3185_<mylut>_numlines` but only a define `MAX31865_<MYLUT>'